src/nfa/operations.cc

```diff
@@ -137,47 +137,43 @@ std::ostream &std::operator<<(std::ostream &os, const Mata::Nfa::Trans &trans) {
     return os << result;
 }
 
-bool Mata::Nfa::are_state_disjoint(const Nfa& lhs, const Nfa& rhs)
-{ // {{{
-    // fill lhs_states with all states of lhs
+bool Mata::Nfa::are_state_disjoint(const Nfa& lhs, const Nfa& rhs) {
```

Comment:
Not relevant to this PR, but this function needs a comment. What is it doind?

Reply:
You need to look at the header file to find documentation for functions. The documentation says:
> Do the automata have disjoint sets of states?

Is this enough of a description?

Reply:
No it is not, because it sounds as a nonsense. States of every automaton are number from 0 to n, the only way how they can be disjoint is that one of them has no states.

Reply:
You are right. This is a relict from Vata. I look at the implementation now. The function checks whether the state number in one automaton is present in the second automaton. With Mata, this function is meaningless, except for the empty set example. Good catch. I will remove this function entirely, then. Agreed?

Reply:
Aha, I thoughgt it was someting that was donng something, just secretly :). Ok.

Reply:
Removed.